src/client/contract-client.js

```diff
@@ -143,11 +163,11 @@ class Client {
    * client as if they were normal functions and return promises that return
    * the end result of the readonly request.
    *
-   * Translate the args to XDR calls -> send a readonly request to the
-   * Mazzaroth node specified in the node-client -> parse and return the
-   * response.
-   *
-   * transaction to the network -> return the result.
+   * 1. Translate the args to XDR calls.
+   * 2. Send a readonly request to the Mazzaroth node specified in the node-client.
+   * 3. Send the transaction to the network.
```

Comment:
I think "3. Send the transaction to the network." can be removed for this function since this is just a less specific duplication of what is described on step 2 here.

Reply:
Totally. Thanks.